sdcm/utils/version_utils.py

```diff
@@ -22,7 +22,7 @@
 PRIMARY_XML_GZ_REGEX = re.compile(r'="(.*?primary.xml.gz)"')
 
 REPOMD_XML_PATH = "repodata/repomd.xml"
-SCYLLA_URL_REGEX = re.compile(".*(?P<url>http.*scylladb.com.*).*")
+SCYLLA_URL_REGEX = re.compile(r".*(?P<url>http.*(=?scylladb.com|//([0-9]{1,3}\.){3}[0-9]{1,3}).*).*")
```

Comment:
Handling only IP addresses isn't enough, for I'm using something like ngrok and such, I'll have a dns name for it.

```suggestion
SCYLLA_URL_REGEX = re.compile(r".*(?P<url>http?://.*")
```
if it's a valid http, you can follow it up, if there are other case which you don't want to follow, for some reason, put them in the code.

if you thing you care only for specific sections each time, so read the file, and don't use regex for this one. might be easier to handle.

Reply:
I removed this validation